pkg/services/ngalert/api/api_ruler.go

```diff
@@ -530,6 +530,7 @@ func toGettableExtendedRuleNode(r ngmodels.AlertRule, provenanceRecords map[stri
 			Provenance:           apimodels.Provenance(provenance),
 			IsPaused:             r.IsPaused,
 			NotificationSettings: AlertRuleNotificationSettingsFromNotificationSettings(r.NotificationSettings),
+			Record:               ApiRecordFromModelRecord(r.Record),
```

Comment:
Always allow reading of recording rules regardless of feature toggle.

If we don't allow this, things get really weird in terms of consistency/ordering if you enable the toggle, create a RR, then disable the toggle. A more advanced solution seemed pointless as the toggle will go away in the future.